src/test/java/com/jcraft/jsch/JSchTest.java

```diff
@@ -64,6 +68,26 @@ void checkLoggerBehavior() throws Exception {
     assertSame(JSch.DEVNULL, jsch.getInstanceLogger(), "instance logger should be DEVNULL");
   }
 
+  @Test
+  void getConfigKeys() throws Exception {
```

Comment:
Can we rename this test method, since it is now testing a method named `getConfig()` and not `getConfigKeys()`?